lib/images/Makefile

```diff
@@ -0,0 +1,39 @@
+# Copyright (C) 2020 Tasos Alvas <tasos.alvas@qwertyuiopia.com>
+# Copyright (C) 2020 DebOps <https://debops.org/>
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+# DebOps Logo Makefile
+# See https://docs.debops.org/en/master/meta/logo.html for build requirements
+
+BUILDDIR = .
+SRCDIR = src
+
+SVG_TARGETS = $(patsubst $(SRCDIR)/%-src.svg,$(BUILDDIR)/%.svg,$(wildcard $(SRCDIR)/*-src.svg))
+
+.PHONY: clean all
+all: debops.png debops-text.png debops-small.png favicon.ico $(SVG_TARGETS)
+
+# Optimized SVG: Use these unless forced to do otherwise
+$(BUILDDIR)/%.svg: $(SRCDIR)/%-src.svg
+	svgo $< -o $@
+
+# PNG with text
+debops-text.png: $(SRCDIR)/debops-src.svg
+	inkscape $< -h 200 -o $@
+
+# PNG without text -- unfortunately grabbed by object ID, not layer name :(
+debops.png: $(SRCDIR)/debops-src.svg
+	inkscape $< -j layer9 -h 200 -o $@
```

Comment:
This needs to change to:
```
debops.png: $(SRCDIR)/debops-src.svg
	inkscape $< -j --export-id layer9 -h 200 -o $@
```
Otherwise inkscape (the version in buster-backports) will fail with:
```
ConcreteInkscapeApplication<Gtk::Application>::on_open: Can't use '--export-filename' with multiple input files (output file would be overwritten for each input file). Please use '--export-type' instead and rename manually.
```